iroh-dns-server/Cargo.toml

```diff
@@ -59,7 +59,7 @@ z32 = "1.1.1"
 
 [dev-dependencies]
 criterion = "0.5.1"
-hickory-resolver = "=0.25.0-alpha.5"
+hickory-resolver = "0.25"
```

Comment:
lets make it `0.25.1` please, so we don't immediately run into issues with minimal versions again

Reply:
Technically `0.25.0` also works. I'll elaborate the bound though!